portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,36 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <h1>Abi's Portfolio</h1>
+        <button onclick="addRandomQuote()">Click here to see a random quote!</button>
+        <div id="greeting-container">]</div>
+        <h2>About Me</h2>
+        <p>My name is Abirami Patchaiyappan, Abi for short. I am currently a sophmore at UC Santa Cruz, studying computer science and cognitive science. I love arts and creativity.</p>
+        <div id=me>
+          <p>I'm from San Jose and have lived in the Bay Area all my life. Here's a picture of me at Santana Row, San Jose.</p>
+          <img src = '/images/Abi.jpg'>
```

Comment:
```suggestion
          <img src="/images/Abi.jpg">
```

Nit: keep the styling consistent in the file, since you've already used double quotes and no spaces around the equal sign elsewhere (see div id, button onclick, etc)